rpi/mode/follow_no_vision.py

```diff
@@ -0,0 +1,36 @@
+import cv2 as cv
+
+from rosa import Rosa
+from rosa.vision.line_tracking import get_line_center
+
+import time
+
+def set_speed(rosa, ls, rs):
+    rosa.left_wheel.speed = ls
+    rosa.right_wheel.speed = rs
+
+
+if __name__ == '__main__':
+    rosa = Rosa('rosa.local')
+
+    while True:
+        reflected = rosa.ground_reflected
+
+        print(reflected)
+        SPEED = 0.1
+        
+        threshold = 300
+        if reflected[0] > threshold and reflected[1] < threshold:
+            print("right")
+            # set_speed(rosa,0,SPEED)
+        elif reflected[0] < threshold and reflected[1] > threshold:
+            print("left")
+            # set_speed(rosa,SPEED, 0)
+        elif reflected[0] < threshold and reflected[1] < threshold:
+            print("straigh")
+            # set_speed(rosa,SPEED, SPEED)
```

Comment:
@EtienneSchmitz When using the two Thymio ground sensors it is often more robust to follow the *edge* of the path, going straight when the sensors have different values, and turning when both sensors have the same value. This avoids two troublesome behaviors on wide paths: oscillating from side to side, and crossing at a shallow angle.

Turning is actually small adjustments to the relative speeds, proportional to the speed rather than absolute values, so that the basic speed can be modified.